### PR TITLE
Reworked translations and added missing texts

### PR DESCRIPTION
**Pull request by kosmas58**
## What:
Added missing translations using [BabelEdit](https://www.codeandweb.com/babeledit).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,8 @@
     "core-js": "^2.5.4",
     "hammerjs": "^2.0.8",
     "rxjs": "^6.3.3",
+    "jquery": "^3.3.1",
+    "popper.js": "^1.14.3",
```
Comment (tomastrajan) [inline package.json]: Why were these dependencies added ?

File: package.json [hunk 2]
```diff
@@ -89,7 +91,6 @@
     "codelyzer": "~4.5.0",
     "cypress": "^3.1.0",
     "express": "^4.16.4",
-    "husky": "^1.1.2",
```
Comment (tomastrajan) [inline package.json]: Why was `husky` removed ?

File: src/assets/i18n/de.json [hunk 3]
```diff
@@ -1,59 +1,95 @@
...
-  "anms.settings.themes.placeholder": "Farbthema",
-  "anms.settings.themes.blue": "Blau",
-  "anms.settings.themes.light": "Hell",
-  "anms.settings.themes.nature": "Natur",
-  "anms.settings.themes.dark": "Dunkel",
-  "anms.settings.themes.night-mode": "Auto Nachtmodus (von 21:00 bis 7:00)",
-  "anms.settings.animations": "Animationen",
-  "anms.settings.animations.page": "Navigation ganzer Seitenübergang",
-  "anms.settings.animations.elements": "Navigation Seitenelemente nach oben schieben"
+	"anms": {
```
Comment (tomastrajan) [inline src/assets/i18n/de.json]: I am against nested properties for translations, in components we use them as for example `anms.features.lazyloading.description`, we want them in same format also in the translation files for the convenience...

**Comment by kosmas58** [comment]
Hey @tomastrajan !

I will follow your suggestions, close this request and create a new one.

**Pull request closed without merging**